src/Cli/dotnet/commands/dotnet-tool/run/LocalizableStrings.resx

```diff
@@ -121,7 +121,7 @@
     <value>Cannot find a tool in the manifest file that has a command named '{0}'.</value>
   </data>
   <data name="CommandDescription" xml:space="preserve">
-    <value>Run local tool.</value>
+    <value>Run local tool. Note that this command cannot be used to run a global tool. </value>
```

Comment:
```suggestion
    <value>Run a local tool. Note that this command cannot be used to run a global tool. </value>
```

Your string looks good, but the existing one could be improved in the meantime 😄